src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import { GlobalStyle } from './styles/GlobalStyles'
 import Theme from './styles/Theme'
+import Button from './components/Button'
```

Comment:
was button used in this file?

Reply:
f1a722aceabae134b3efe420c4d8249fd0023fc6